src/automation/README.md

```diff
@@ -0,0 +1,29 @@
+Automation tools for running observation inventory utils. Runs as a separate script from the main source code accessing the main code via CLI calls. 
```

Comment:
```suggestion
Automation tools for running observation inventory utilities. Runs as a separate script from the main source code accessing the main code via CLI calls. 
```

Reply:
I think using utils makes more sense here as it is a reference to the code package name